docker-compose.yml

```diff
@@ -229,14 +233,13 @@ services:
     <<: *app
     environment:
       - CONFDIR=/app/samvera/hyrax-webapp/solr/conf
+      - PATH=/app/samvera/hyrax-webapp/bin:${PATH}
```

Comment:
For some reason I couldn't get the echo to set it correctly in the Dockerfile. 🤷🏼‍♀️